js/slider.js

```diff
@@ -0,0 +1,85 @@
+let arrayOfSliderProd;
+fetch('./jsons/slider.json').then(respons => respons.json()).then(data => {arrayOfSliderProd = data; createSlider(); changSlide()});
```

Comment:
response

Reply:
and please makeeach step on separate line.add prettier.

Reply:
when you call function - this function should be close as soon as possible to place where it called.

Reply:
`fetch('./jsons/slider.json')
  .then(respons => respons.json())
  .then(data => {
     arrayOfSliderProd = data;
     createSlider(); 
     changSlide()
});`